integration_tests/nlg_meta_eval_test.py

```diff
@@ -36,6 +36,6 @@ def test_da_cs_en(self):
         processor = get_processor_class(TaskType.nlg_meta_evaluation)()
 
         sys_info = processor.process(metadata, data)
-
-        self.assertGreater(len(sys_info.results.analyses), 0)
-        self.assertGreater(len(sys_info.results.overall), 0)
+        self.assertAlmostEqual(
```

Comment:
I think the original assertion does some different checks, and needs to be maintained as well.